<?xml version="1.0"?>
<launch>
    <group ns="honey">
        <rosparam file="$(find reswarm_dmpc)/config/leader_iface.yaml" command="load" ns="" />
        <node pkg="reswarm_dmpc" type="ros_node_leader.py" name="leader_iface" output="screen">
            <!-- Topics -->
            <remap from="~pose_topic" to="loc/pose"/>
            <remap from="~twist_topic" to="loc/twist"/>
            <remap from="~follower_pose" to="/bumble/loc/pose"/>
            <remap from="~control_topic" to="gnc/ctl/command"/>
            <remap from="~broadcast_information" to="information"/>
            <remap from="~flight_mode" to="mob/flight_mode"/>
            <!-- Services -->
            <remap from="~get_control_srv" to="get_control"/>
            <remap from="~set_weights_srv" to="set_weights"/>
            <remap from="~onboard_ctl_enable_srv" to="gnc/ctl/enable"/>
            <remap from="~pmc_timeout_srv" to="hw/pmc/set_timeout"/>
            <remap from="~start_srv" to="start"/>
        </node>
        <node pkg="reswarm_dmpc" type="leader_ctl_node" name="leader_ctl_node" output="log">
            <remap from="get_control_srv" to="get_control"/>
            <remap from="set_weights_srv" to="set_weights"/>
        </node>    
    </group>
</launch>